lib/i18n/index.ts

```diff
@@ -0,0 +1,67 @@
+import { po } from 'gettext-parser';
+import Gettext from 'node-gettext';
+import { logger } from '../logger';
+import { readSystemFile } from '../util/fs';
+
+const gt = new Gettext();
+
+export function gettext(msgid: string): string {
```

Comment:
Did you consider naming the function 'getText'?

Reply:
@astellingwerf I prefer to keep the traditional function name of Gettext, actually Gettext provided a set of keyword specifications for `xgettext` promgram, which look for translatable strings for source files by.

Please take look at the [manual](https://www.gnu.org/software/gettext/manual/gettext.html#Choice-of-input-file-language), for JavaScript folllowing functions are expected: _, gettext, dgettext:2, dcgettext:2, ngettext:1,2, dngettext:2,3, pgettext:1c,2, dpgettext:2c,3. 

`_` looks like a nice shortcut, I will export this function to decline the cost of typing.